src/features/unified-share-modal/UnifiedShareForm.js

```diff
@@ -706,6 +709,13 @@ class UnifiedShareForm extends React.Component<USFProps, State> {
             !showCollaboratorList &&
             allShareRestrictionWarning;
 
+        const showContentInsightsToggle =
+            onAdvancedContentInsightsToggle &&
```

Comment:
Parts of this conditional are the exact same as the ones in the statement above. I'm guessing that combination of conditions has greater meaning in the context of the form than they do individually with regard to whether or not we want to show those elements. Do you think it would be a good idea to extract those common conditions out into their own function? It might make the code more readable and easy to understand.

Reply:
Hey @bfoxx1906 , @tjuanitas . I added some logic regarding this comment @bfoxx1906 added. I agree with him that it would be better to make that condition more readable. PTAL and please fell free to share your thoughts

Reply:
Hi @tjuanitas and @bfoxx1906 . Bumping this to get final feedback for this change. Thanks